net/tmate-ssh-server/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tmate-ssh-server
+PKG_VERSION:=511fd2bd852464e76824279609a34ee93fe148a4
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/tmate-io/tmate-ssh-server/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=68602496ca6a17ea80f5be53eba047897ac714c7cdfcb3bcdc12c56b8f3c3b45
+
+PKG_LICENSE:=ISC
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+
+PKG_FIXUP:=autoreconf
+PKG_BUILD_DEPENDS:=ncurses
```

Comment:
Are you sure this is needed as you are already depending on `libncurses`?
Or did you rather mean to refer to `ncurses/host`?

Reply:
Thanks I'll look into that

Reply:
Works without as well, thank you.